tests/modules/programs/git/git-difftastic.nix

```
@@ -0,0 +1,20 @@
+{ ... }:
+{
```

```suggestion
{
```

nit(not blocking): Can just omit when not using them.